contents/tracks.mdx

```diff
@@ -4,100 +4,225 @@ showTitle: false
+    subtitle="Building a startup isn't easy. But we can put you on the right track!"
 />
+<h3 className="text-xl m-0 font-semibold opacity-75 text-center !pb-8">
+Tracks is a curated reading list for founders building product-led companies like PostHog. We build in the open, so we're happy to share all the details of how we've got this far, and the lessons we learned on the way.</h3>
 
-PostHog Tracks is a curated course of tutorials and other lessons which cover common uses for particular roles, as well as general advice for all users - and it's constantly expanding!
+<h3 className="mb-0">How to <span style='color: #f64e00;'>hire a team</span> of world-class talent</h3>
 
-<p class="mt-4 pb-12 flex space-x-2 items-center justify-center">
-<CallToAction href="/tutorials">Browse all tutorials</CallToAction> 
-<CallToAction href="/questions" type="outline">Ask a question</CallToAction>
-</p>
-
-<h3 className="mb-0">PostHog 101</h3>
-
-Want a quick summary of the essential info? Look no further than PostHog 101!
+It all starts with getting the right people on your project. You can check the handbook for more information about our [small team structure](/handbook/company/small-teams) and [minimal approach to management](/handbook/company/management), or head to [our staff profiles](/team) to understand what our team looks like now.
+<br />
 
 <TutorialsSlider slugs={[
-  "/tutorials/non-technical-guide-to-data",
-  "/tutorials/event-tracking-guide",
-  "/tutorials/regex-basics",
-  "/tutorials/identifying-users-guide",
-  "/tutorials/next-steps-after-installing"
+  "/blog/posthog-first-five",
+  "/blog/how-we-do-hiring-and-hr-at-posthog",
+  "/blog/early-stage-startup-hiring-strategy",
+  "/blog/meetings",
+  "/blog/startup-job-titles"
 ]} />
 
 <br />
 
-<h3 className="mb-0">PostHog for <span className="text-red">product engineers</span></h3>
-
-Are you part-engineer, part-product person? This is the lesson track for you!
+<h4 class="text-center">Some of the books we recommend to each other (and you)</h4>
+<Section cols={3}>
+    <div>      
+    <a href="https://www.goodreads.com/book/show/54736620-ask-iwata">
+        <GatsbyImage image={getImage(props?.images[0])} alt="Ask Iwata" objectFit="contain" style={{ maxHeight: 200 }} class="center" />
+    </a>
+        <p class="text-center">
+          Recommended by <a href="/community/profiles/59">Joe Martin</a>
+        </p>
+    </div>
+    <div>
+    <a href="https://www.goodreads.com/book/show/60695040-from-switzerland">
+        <GatsbyImage image={getImage(props?.images[11])} alt="From Switzerland" objectFit="contain" style={{ maxHeight: 200 }} class="center" />
+    </a>
+        <p class="text-center">
+          Recommended by <a href="/community/profiles/227">Lottie Coxon</a>
+        </p>
+    </div>
+    <div>
+    <a href="https://www.goodreads.com/book/show/42348376-secrets-of-sand-hill-road">
```

Comment:
These two don't have a lot to do with hiring 🤔

Reply:
Haha, yeah, nobody suggested any books about hiring. We can sub something in from Coua or Grace if needed, but for now this is just stuff from the _people_.